doc/source/examples/multiphysics/bubble-detachment-shear-flow/bubble-detachment-shear-flow.rst

```diff
@@ -0,0 +1,417 @@
+Below, all the parameters are set for a simulation whose shear rate :math:`S = 500 \ \text{s}^{-1}`. Detailed instructions on how to generate the parameters files automatically are given in the "Running the Simulation" section of this example.
+
+-----------------
+Parameter File
+-----------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+Time integration is handled by a 2nd order backward differentiation scheme (`bdf2`), for a :math:`0.5 \ \text{s}` simulation time with an initial time step of :math:`5 \times 10^{-7} \ \text{s}`. Time-step adaptation is enabled using ``adapt = true`` and the max CFL is :math:`0.9`. ``output boundaries`` is set to ``true`` to get a ``.vtu`` file containing the indices of the boundaries of the domain. The maximum time step is computed using the capillary time step condition given below:
+
+.. math::
+    \Delta t < \Delta t_\sigma = \sqrt{\frac{(\rho_a+\rho_l)\Delta x^3}{4\pi\sigma}}
+
+where :math:`\rho_a` and :math:`\rho_l` are the densities of the gas phase and the liquid phase, respectively, :math:`\Delta x` is the characteristic cell size of the mesh and :math:`\sigma` is the surface tension coefficient.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method            = bdf2
+      set output name       = article-3d
+      set output frequency  = 15
+      set output path       = ./outputs/output/
+      set time end          = 0.5
+      set adapt             = true
+      set max cfl           = 0.9
+      set time step         = 5e-7
+      set max time step     = 7.9784e-06 # Capillary timestep condition
+      set output boundaries = true
+    end
+
+
+Multiphysics
+~~~~~~~~~~~~
+
+The ``multiphysics`` subsection is used to enable the ``cahn hilliard`` solver.
+Note that the fluid dynamics are solved by default.
+
+.. code-block:: text
+
+    subsection multiphysics
+      set cahn hilliard = true
+    end
+    
+Dimensionality
+~~~~~~~~~~~~~~
+
+The ``dimensionality`` subsection is used to define the unit length as :math:`0.001 \text{m} = 1 \ \text{mm}`. This setting helps with the convergence of the solver.	
+
+.. Note:: When using the dimensionality parameters, the problem and the physical properties are rescaled using the new units specified by the user. This means that physical properties can be given their value in SI units and will automatically be rescaled. The resulting fields (velocity and pressure for instance) will also be rescaled accordingly. One exception to this is the source terms and the initial conditions, which need to be specified in rescaled units.
+
```

Comment:
I guess also boundary conditions right?
It only rescales the unit for the physical properties in the end.